src/AsmResolver.PE/Debug/CodeView/CodeViewDataSegment.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+
+namespace AsmResolver.PE.Debug.CodeView
+{
+    /// <summary>
```

Comment:
Docs refer to wrapping around an instance of `ISegment`, but implementation does not do that.